walkthroughs/3-low-code-api-development/walkthrough.adoc

```diff
@@ -35,12 +33,15 @@ It also allows you to identify important design decisions prior to coding and de
 .Red Hat OpenShift
 ****
 * link:{openshift-console-url}[Console, window="_blank"]
+* link:https://help.openshift.com/[OpenShift Online Help Center, window="_blank"]
```

Comment:
Might need to reconsider this. It links to OpenShift Online help. I can change to "https://docs.openshift.com/dedicated/3/welcome/index.html" in my PR later

Reply:
(OpenShift Dedicated docs)

Reply:
Yep, agreed that we should look into using a better URL here. For now, I have all the SPs using the same exact resource links for OpenShift. We should update all of them once we know what this should really be.